rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc

```diff
@@ -27,24 +27,26 @@ Machine pools are a higher level construct to compute machine sets.
 
 A machine pool creates compute machine sets that are all clones of the same configuration across availability zones. Machine pools perform all of the host node provisioning management actions on a worker node. If you need more machines or must scale them down, change the number of replicas in the machine pool to meet your compute needs. You can manually configure scaling or set autoscaling.
 
-By default, a cluster has one machine pool. During cluster installation, you can label this machine pool. After the cluster is installed, the `Default` machine pool cannot be deleted, and its node type or size cannot be changed.
+By default, a cluster has one machine pool. During cluster installation, you can define instance type or size and add labels to this machine pool.
 
 After a cluster's installation:
 
-* You can relabel a `Default` machine pool.
+* You can remove or add labels to any machine pool.
 * You can add additional machine pools to an existing cluster.
+* You can add taints to any machine pool as long as there is one machine pool without any taints.
```

Comment:
The machinepool should at the same time have untaint 2(for single zone) and 3(for multi zone) replicas
```
lixue@Xue-Lis-MacBook-Pro ~ % rosa edit machinepool worker -c sdq-ci-twppd --taints kwy=value:NoSchedule
E: Failed to update machine pool 'worker' on cluster 'sdq-ci-twppd': Machine pool 'worker' for cluster '263hpcg0hqb5v4qkp94taut6rnn4apat' can't be updated with provided params. At least one machine pool able to run OCP workload is required. Pool should not have taints and minimum replica count should be '2'
lixue@Xue-Lis-MacBook-Pro ~ % rosa list machinepools -c sdq-ci-twppd                                   
ID      AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS                                                                TAINTS    AVAILABILITY ZONES                    SUBNETS                                                                         SPOT INSTANCES  DISK SIZE  
sz1     No           1         m5.xlarge                                                                                      us-west-2b                            subnet-02c45e21191037c15                                                        No              300 GiB
worker  Yes          3-6       m5.xlarge      label1/openshift.com=labelvalue1, label2/openshift.com=labelvalue2              us-west-2a, us-west-2b, us-west-2c    subnet-0bc1a548e9a8b46e1, subnet-02c45e21191037c15, subnet-09416be08e29bd88c    No              150 GiB
```